### PR TITLE
Created skeleton

### PR DESCRIPTION
**Pull request by jmckenney**
Created skeleton... that's it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (jmckenney) [inline index.html:7]: Make  a real title please.

Comment (JakeDudum) [inline index.html:7]: Make a real title.

Comment (jmckenney) [inline index.html:7]:
```suggestion
    <title>Cool Project</title>
```

**Review by JakeDudum — APPROVED** [review]
Looks good!

**Pull request merged**